index_service/templates/verification/email.html

```diff
@@ -0,0 +1,81 @@
+{% extends "mail_base.html" %}
+{% load i18n %}
+
+{# Actually never used #}
```

Comment:
Email clients normally use the Subject header as the title... I'm not sure this is displayed in any email client. Many HTML mail examples have a \<title>